src/fortran_routing/diffusive_v02frwk/maindiff.f90

```diff
@@ -1,4 +1,4 @@
-module mdiffv2
+module mdiff
```

Comment:
really starting to think this should be explicit `mdiffusive` or `m_diffusive` or even just `module diffusive`.  I'm not sure what the `main` is meant to imply, and it is less ambiguous to use `diffusive` over `diff` throughout.

Reply:
agree - changed the main diffusive model script to simply `diffusive.f90`